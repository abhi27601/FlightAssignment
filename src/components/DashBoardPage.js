import React from "react";
const DashBoardPage = () => (
  <div>
      DashBoardPage content
  </div>


);
console.log("Gauri");

export default DashBoardPage;
