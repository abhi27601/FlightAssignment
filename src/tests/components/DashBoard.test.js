import React from 'react';
import { shallow } from "enzyme";
import DashBoardPage from "../../components/DashBoardPage";

test('should render expensedahboard page', () => {
    const wrapper = shallow(<DashBoardPage />)
    expect(wrapper).toMatchSnapshot();
})