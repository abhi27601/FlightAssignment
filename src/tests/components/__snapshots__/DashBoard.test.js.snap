// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensedahboard page 1`] = `
<div>
  DashBoardPage content
</div>
`;
